Maya dependency fails a pod lint

hello lovlies––I'm trying to update a community library I maintain to support Swift 3 and all, however the `Moya+Alamofire.swift` file is ERROR'ing the lint :( 

here's the output of the `pod lib lint`:
```
➜  Moya-Gloss git:(master) ✗ pod lib lint

 -> Moya-Gloss (2.0.0-beta.2)
    - WARN  | [Moya-Gloss/Core, Moya-Gloss/RxSwift, Moya-Gloss/ReactiveCocoa, and more...] xcodebuild:  Moya/Source/Moya+Internal.swift:206:16: warning: value 'progress' was defined but never used; consider replacing with boolean test
    - ERROR | [Moya-Gloss/Core, Moya-Gloss/RxSwift, Moya-Gloss/ReactiveCocoa, and more...] xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.
    - ERROR | [Moya-Gloss/Core, Moya-Gloss/RxSwift, Moya-Gloss/ReactiveCocoa, and more...] xcodebuild:  Moya/Source/Moya+Alamofire.swift:9:46: error: no type named 'StreamRequest' in module 'Alamofire'
    - WARN  | [Moya-Gloss/RxSwift] xcodebuild:  Moya/Source/RxSwift/Moya+RxSwift.swift:31:20: warning: 'init' is deprecated: renamed to 'Disposables.create'
    - NOTE  | [Moya-Gloss/RxSwift] xcodebuild:  Moya/Source/RxSwift/Moya+RxSwift.swift:31:20: note: use 'Disposables.create' instead
    - WARN  | [Moya-Gloss/RxSwift] xcodebuild:  Moya/Source/RxSwift/Moya+RxSwift.swift:57:20: warning: 'init' is deprecated: renamed to 'Disposables.create'
    - NOTE  | [Moya-Gloss/RxSwift] xcodebuild:  Moya/Source/RxSwift/Moya+RxSwift.swift:57:20: note: use 'Disposables.create' instead
    - WARN  | [Moya-Gloss/RxSwift] xcodebuild:  RxCocoa/RxCocoa/Common/TextInput.swift:74:28: warning: 'RxTextInput' is deprecated: renamed to 'TextInput'
    - NOTE  | [Moya-Gloss/RxSwift] xcodebuild:  RxCocoa/RxCocoa/Common/TextInput.swift:74:28: note: use 'TextInput' instead
    - WARN  | [Moya-Gloss/RxSwift] xcodebuild:  RxCocoa/RxCocoa/Common/TextInput.swift:67:29: warning: 'RxTextInput' is deprecated: renamed to 'TextInput'
    - NOTE  | [Moya-Gloss/RxSwift] xcodebuild:  RxCocoa/RxCocoa/Common/TextInput.swift:67:29: note: use 'TextInput' instead
    - WARN  | [Moya-Gloss/ReactiveCocoa] xcodebuild:  Moya/Source/ReactiveCocoa/Moya+ReactiveCocoa.swift:5:45: warning: 'where' clause next to generic parameters is deprecated and will be removed in the future version of Swift

[!] Moya-Gloss did not pass validation, due to 2 errors and 6 warnings.
You can use the `--no-clean` option to inspect any issue.
```

relevant podspec is [here](https://github.com/spxrogers/Moya-Gloss/blob/master/Moya-Gloss.podspec). i ran with the `--verbose` tag and found the **watchOS 2.0 platform** with the **ReactiveCocoa** subspec that failed and building and removing the watchOS deployment target succeeds the lint. 

:/ hoping maybe some other contributors would have some insight, maybe @ashfurrow?

--- a/Changelog.md
+++ b/Changelog.md
@@ -1,6 +1,7 @@
 # Next
 
 - **Breaking Change** Throw dedicated `Error.jsonMapping` when `mapJSON` fails to parse JSON
+- Removed the unused `StreamRequest` typealias that was causing watchOS failures.
 
 # 8.0.0-beta.1
 

--- a/Source/Moya+Alamofire.swift
+++ b/Source/Moya+Alamofire.swift
@@ -6,7 +6,6 @@ internal typealias Request = Alamofire.Request
 internal typealias DownloadRequest = Alamofire.DownloadRequest
 internal typealias UploadRequest = Alamofire.UploadRequest
 internal typealias DataRequest = Alamofire.DataRequest
-internal typealias StreamRequest = Alamofire.StreamRequest
 
 internal typealias URLRequestConvertible = Alamofire.URLRequestConvertible
 

